### PR TITLE
Style Changes for Issue #195

### PR DESCRIPTION
**Pull request by drownedout**
Changes for:

 'Submit pledge' button should be BIG (super button) and should have more blank space above it, separating it from the left nav menu items.
 Add lines into the page separating top entry, left nav and main right window.
 Black bar at the top should be gray, type should be black

### CODE REVIEW

**Comment by colinxfleming** [comment]
Paging @mebates . Here are the changes as they render for me. Do these pass muster from yr perspective? 

screen shot 2016-04-10 at 2 13 57 am

File: app/views/pregnancies/_call_log.html.erb [hunk 1]
```diff
@@ -8,6 +8,10 @@
 			<th>Result</th>
 			<th>CM</th>
 		</tr>
+		<tbody>
```
Comment (colinxfleming) [inline app/views/pregnancies/_call_log.html.erb]: Can you revert changes to these four lines? Call logs are handled in the loop below this.

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -192,3 +203,6 @@ section {
     }
   }
 }
+
+@import 'patients';
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss]: The rails asset pipeline automatically loads these files, so we don't need to double-include them here. (See section 2.1 at http://guides.rubyonrails.org/asset_pipeline.html, specifically the stuff about `require_tree`.) 

Can you remove these two imports?

**Comment by colinxfleming** [comment]
Thanks much for your work on this @drownedout ! Had a few comments, but this looks pretty good to my eyes. After resolution of the comments and @mebates' confirmation I'll merge it in. 

:100:

**Comment by colinxfleming** [comment]
Bumps #195, specifically: increasing size of submit pledge button, adding lines, and changing the black bar at the top.

**Comment by colinxfleming** [comment]
@drownedout I got confirmation from @mebates that these look good. Can you do me a solid and resolve the merge conflicts, and make the edits I suggested? After that we're good to go. 

Thanks much bud!

**Comment by drownedout** [comment]
Where do I locate the merge conflicts and comments? If it's too much trouble right now I can fix them up tomorrow and polish off some more front-end stuff.

**Comment by colinxfleming** [comment]
@drownedout You can see the comments here on this page (specifically, you'll want to note the line comments on the code), but you can see them highlighted more clearly here:

screen shot 2016-04-17 at 12 23 48 pm

To resolve merge conflicts, run the following commands:

`git pull origin master # pull down the changes`
`git status # see what files the conflicts are in`
 Then open up the files in a text editor and resolve in such a way that keeps your changes. (The changes will be separated with a bunch of <s and =s.) Probably most of them are going to be pretty trivial. Let me know if you have questions.

**Comment by drownedout** [comment]
Got it.

I'll try and get those fixed before C4DC

**Pull request merged**